CircleciScripts/run_integrationtest.py

```diff
@@ -5,37 +5,38 @@
 test_results = sys.argv[1]
 root = sys.argv[2]
 credentials = sys.argv[3]
+testmodule =  sys.argv[4]
 
-testmodules =  ["aws-android-sdk-apigateway-test",
-                "aws-android-sdk-autoscaling-test",
-                "aws-android-sdk-cloudwatch-test",
-                "aws-android-sdk-elb-test",
-                "aws-android-sdk-ddb-test",
-                "aws-android-sdk-ddb-mapper-test",
-                "aws-android-sdk-comprehend-test",
-                "aws-android-sdk-iot-test",
-                "aws-android-sdk-kinesis-test",
-                "aws-android-sdk-kinesisvideo-archivedmedia",
-                "aws-android-sdk-lambda-test",
-                "aws-android-sdk-mobile-client",
-                "aws-android-sdk-rekognition-test",
-                "aws-android-sdk-polly-test",
-                "aws-android-sdk-s3-test",
-                "aws-android-sdk-sdb-test",
-                "aws-android-sdk-sqs-test",
-                "aws-android-sdk-sns-test",
-                "aws-android-sdk-ses-test",
-                "aws-android-sdk-transcribe-test",
-                "aws-android-sdk-translate-test",
-               ]
+ignoreFailureList = {
```

Comment:
Nit : can be renamed to ignored_failures_dict

Reply:
done